packages/material-ui/src/Link/Link.d.ts

```diff
@@ -11,6 +11,7 @@ export interface LinkTypeMap<P = {}, D extends React.ElementType = 'a'> {
        */
       children?: React.ReactNode;
       /**
+       * Override or extend the styles applied to the component.
```

Comment:
In case CI comes back green we should investigate why this doesn't affect the docs output. It should actually affect documentation. Otherwise the documentation might be generated from hardcoded values.

Reply:
So ignored I guess.

Reply:
"ignored" as the documentation ignores it? It might be coming from https://github.com/mui-org/material-ui/blob/7e9943e9a00f04ce44741c292f7ac051b655b01f/packages/material-ui-styles/src/withStyles/withStyles.d.ts#L117

I don't think that it matters, as far as I understand, until it doesn't create an issue for developers, it's not a problem.

Reply:
So you agree, that the change wasn't necessary. I just checked and IntelliSense works without this change.

Reply:
> "ignored" as the documentation ignores it? It might be coming from

No, you ignored this review.

Reply:
Definitely, the objective is to match the other components, not to solve the root cause. If it has uncovered something, great.